src/tcgwars/logic/impl/gen8/DarknessAblaze.groovy

```diff
@@ -0,0 +1,3394 @@
+          energyCost F, C, C
+          attackRequirement {}
+          onAttack {
+            damage 80
+          }
+        }
+        move "Heavy Rock Cannon", {
+          text "210 damage. This Pokémon can’t use this attack during your next turn."
+          energyCost F, C, C, C
+          attackRequirement {}
+          onAttack {
+            damage 210
+          }
+        }
+      };
+      case DIGGERSBY_99:
+      return evolution (this, from:"Bunnelby", hp:HP130, type:F, retreatCost:3) {
+        weakness G
+        move "Hammer In", {
+          text "90 damage. "
+          energyCost F, C, C
+          attackRequirement {}
+          onAttack {
+            damage 90
+          }
+        }
+        move "Land Crush", {
+          text "140 damage. "
+          energyCost F, F, C, C
+          attackRequirement {}
+          onAttack {
+            damage 140
+          }
+        }
+      };
+      case ZYGARDE_100:
+      return basic (this, hp:HP140, type:F, retreatCost:3) {
+        weakness G
+        move "Bite", {
+          text "30 damage. "
+          energyCost C
+          attackRequirement {}
+          onAttack {
+            damage 30
+          }
+        }
+        move "Power Blast", {
+          text "130 damage. Discard an Energy attached to this Pokémon."
+          energyCost F, F, C, C
+          attackRequirement {}
+          onAttack {
+            damage 130
+          }
+        }
+      };
+      case PASSIMIAN_101:
+      return basic (this, hp:HP110, type:F, retreatCost:1) {
+        weakness P
+        move "Call for Family", {
+          text "Search your deck for up to 2 Basic Pokémon and put them on your Bench. Then, shuffle your deck."
+          energyCost C
+          attackRequirement {}
+          onAttack {
+
+          }
+        }
+        move "Rock Hurl", {
+          text "70 damage. This attack’s damage isn’t affected by Resistance."
+          energyCost F, C, C
+          attackRequirement {}
+          onAttack {
+            damage 70
+          }
+        }
+      };
+      case GRIMER_102:
+      return basic (this, hp:HP080, type:D, retreatCost:3) {
+        weakness F
+        move "Stomp Off", {
+          text "Discard 1 card from the top of your opponent’s deck."
+          energyCost D
+          attackRequirement {}
+          onAttack {
+
+          }
+        }
+        move "Sludge Whirlpool", {
+          text "50 damage. "
+          energyCost D, C, C
+          attackRequirement {}
+          onAttack {
+            damage 50
+          }
+        }
+      };
+      case MUK_103:
+      return evolution (this, from:"Grimer", hp:HP140, type:D, retreatCost:4) {
+        weakness F
+        move "Triple Poison", {
+          text "Your opponent’s Active Pokémon is now Poisoned. Put 3 damage counters for this Poison between turns."
+          energyCost D
+          attackRequirement {}
+          onAttack {
+
+          }
+        }
```

Comment:
Look to be missing the second move for Muk too

[Muk](https://www.pokebeach.com/news/0420/038109_P_BETOBETON-1.jpg)